README.rst

```diff
@@ -154,6 +149,117 @@ Usage
    the changes made to it will be visible for both the Docker
    container and the Windows system.
 
+#.  Check the shared folder :
+
+    - run command
+
+    .. code:: sh
+
+	ls
+
+    - the docker_shared_folder should be highlighted in green : 
```

Comment:
Should probably have an empty line after, otherwise it may be ambiguous to the rst parser.